storm-client/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -1907,4 +1909,117 @@ private void readArchive(ZipFile zipFile) throws IOException {
             }
         }
     }
+
+    /**
+     * Create a map of forward edges for bolts in a topology. Note that spouts can be source but not a target in
+     * the edge. The mapping contains ids of spouts and bolts.
```

Comment:
Comments  here and below need to be updated since the mapping only contains bolts.

Reply:
Key can be bolt or spout. Value contains bolt ids only.

Reply:
okay